plugin/src/main/java/io/jenkins/plugins/remotingkafka/GlobalKafkaConsumerConfiguration.java

```diff
@@ -0,0 +1,59 @@
+package io.jenkins.plugins.remotingkafka;
+
+import hudson.Extension;
+import hudson.util.FormValidation;
+import jenkins.model.GlobalConfiguration;
+import org.apache.commons.lang.StringUtils;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.QueryParameter;
+
+import java.util.Properties;
+
+/**
+ * Global configuration for Kafka(1.1.0) consumer, according to Kafka documentation
+ * https://kafka.apache.org/documentation/#consumerconfigs.
+ */
+@Extension
```

Comment:
For https://issues.jenkins-ci.org/browse/JENKINS-51471 it will be needed to add `@Symbol` here: https://issues.jenkins-ci.org/browse/JENKINS-51471 . Fine for now